pl/tutorials/tour/mixin-class-composition.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: tutorial
+title: Kompozycja domieszek
+
+disqus: true
+
+tutorial: scala-tour
+num: 5
+language: pl
+tutorial-next: anonymous-function-syntax
+tutorial-previous: traits
+---
+
+W przeciwieństwie do języków które wspierają jedynie pojedyncze dziedziczenie, Scala posiada bardziej uogólniony mechanizm ponownego wykorzystania klas. Scala umożliwia na wykorzystanie _nowych elementów klasy_ (różnicy w stosunku do klasy bazowej) w definicji nowej klasy. Wyraża się to przy pomocy _kompozycji domieszek_.
```

Comment:
`domieszka` is a Polish term for `mixin` (see [Wikipedia](https://pl.wikipedia.org/wiki/Domieszka_(programowanie_obiektowe)) page) which is something different than just `trait`.